src/index.html

```diff
@@ -11,12 +11,104 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="go_somewhere"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="some pic"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="go_somewhere"
```

Comment:
The same, it's not valid at all, I get 404 in this case
<img width="1436" alt="image" src="https://github.com/user-attachments/assets/dec8f80b-c337-4045-8405-32003ff3d78e">

You can point to some anchor on the page, for example `#apple`